gen/struct_test.go

```diff
@@ -1193,6 +1193,56 @@ func TestStructGoTags(t *testing.T) {
 	assert.Equal(t, `json:"foobarWithRequired,required"`, string(foobarWithRequired.Tag))
 }
 
+func TestNotOmitEmpty_GoTagBehavior(t *testing.T) {
+	notOmitemptyStruct := &ts.NotOmitEmpty{
+		NotOmitEmptyString: nil,
+		NotOmitEmptyInt:    nil,
+		NotOmitEmptyBool:   nil,
+		NotOmitEmptyList:   nil,
+		NotOmitEmptyMap:    nil,
+	}
+
+	// assert that when the !omitempty tag is present in thrift, omitempty is not added to the go tags
+	notOmitEmptyString, _ := reflect.TypeOf(notOmitemptyStruct).Elem().FieldByName("NotOmitEmptyString")
```

Comment:
Can we pull the repeated part here into a local variable.

```suggestion
	structT := reflect.TypeOf(ts.NotOmitEmpty{})
	notOmitEmptyString, _ := structT.FieldByName("NotOmitEmptyString")
```